File: src/main/java/net/usbwire/usbplus/commands/ConfigCommand.java
package net.usbwire.usbplus.commands;

import org.incendo.cloud.annotations.Command;
import org.incendo.cloud.annotations.processing.CommandContainer;
import gg.essential.vigilance.gui.SettingsGui;
import net.usbwire.usbplus.USBPlus;
import net.usbwire.usbplus.config.Config;
import net.usbwire.usbplus.util.Util;

@CommandContainer
public class ConfigCommand {
	public ConfigCommand() {
	}

	@Command("usbplus")
	public void handle() {
		Config.openConfig();
	}
}

File: src/main/kotlin/net/usbwire/usbplus/commands/ConfigCommand.kt
package net.usbwire.usbplus.commands

import gg.essential.api.commands.*
import gg.essential.api.utils.GuiUtil
import net.usbwire.usbplus.USBPlus
import net.usbwire.usbplus.config.Config

object ConfigCommand : Command(USBPlus.name.lowercase(), false) {
	@DefaultHandler
	fun handle() {
		val gui = Config.gui() ?: return
		GuiUtil.open(gui)
	}
}
